tests/cypress/e2e/copyCutPaste.cy.ts

```diff
@@ -58,70 +76,68 @@ describe('Copy Cut and Paste tests with page composer', () => {
-        let contextMenu = composer.openContextualMenuOnLeftTree('About');
+        let contextMenu = composer.openContextualMenuOnLeftTree('Investors');
         contextMenu.copy();
         contextMenu = composer.openContextualMenuOnLeftTreeUntil('Newsroom', 'Paste');
         contextMenu.paste().then(() => {
-            checkDescriptions('/sites/digitall/home/newsroom/about');
-            getNodeByPath('/sites/digitall/home/newsroom/about', ['jcr:title'], 'en').then(result => {
-                expect(result?.data?.jcr?.nodeByPath?.name).to.eq('about');
-                expect(result?.data?.jcr?.nodeByPath?.properties[0].value).to.eq('About');
+            checkDescriptions('/sites/digitall/home/newsroom/investors');
+            getNodeByPath('/sites/digitall/home/newsroom/investors', ['jcr:title'], 'en').then(result => {
+                expect(result?.data?.jcr?.nodeByPath?.name).to.eq('investors');
+                expect(result?.data?.jcr?.nodeByPath?.properties[0].value).to.eq('Investors');
             });
-            deleteNode('/sites/digitall/home/newsroom/about');
+            deleteNode('/sites/digitall/home/newsroom/investors');
         });
         cy.logout();
     });
 
-    // This test was timing out on the waitUntil.
-    // To be fixed in BACKLOG-22510
-    it.skip('Cut and paste under another site / check vanity url isn\'t the same', () => {
+    it('Cut and paste under another site / check vanity url isn\'t the same (our companies page)', () => {
         const composer = new PageComposer();
         cy.login();
         PageComposer.visit('digitall', 'en', 'home.html');
-        let contextMenu = composer.openContextualMenuOnLeftTree('About');
+        let contextMenu = composer.openContextualMenuOnLeftTree('Our Companies');
         contextMenu.cut();
-        composer.openContextualMenuOnLeftTreeUntil('Newsroom', 'Paste');
+        cy.wait(5000);
         PageComposer.visit('testsite', 'en', 'home.html');
         contextMenu = composer.openContextualMenuOnLeftTreeUntil('Home', 'Paste');
         contextMenu.paste().then(() => {
             cy.waitUntil(
                 () =>
-                    getVanityUrl('/sites/testsite/home/about', ['en']).then(
-                        result => result?.data?.jcr?.nodeByPath?.vanityUrls[0]?.url === '/about'
+                    getVanityUrl('/sites/testsite/home/our-companies', ['en']).then(
+                        result => result?.data?.jcr?.nodeByPath?.vanityUrls[0]?.url === '/our-companies'
                     ),
                 {
-                    timeout: 30000,
-                    interval: 1000,
+                    timeout: 60000,
+                    interval: 2000,
                     verbose: true
                 }
             );
-            moveNode('/sites/testsite/home/about', '/sites/digitall/home', 'about');
+            moveNode('/sites/testsite/home/out-companies', '/sites/digitall/home', 'out-companies');
```

Comment:
hi, is this a typo? should it be `our-companies` ? also I'm guessing it's passing because there's no check done after the node, and I'm not really sure why it's here as it's doing a move through a graphql call so not really testing page composer. maybe we should just remove.

Reply:
I just realized it's for restoring the page back to digitall (as it's a cut), but maybe not used afterwards. so we should keep and just fix the typo.